src/libpgmoneta/prometheus.c

```diff
@@ -947,6 +994,124 @@ general_information(int client_fd)
    }
    data = pgmoneta_append(data, "\n");
 
+   data = pgmoneta_append(data, "#HELP pgmoneta_server_timeline The current timeline a server is on\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_server_timeline counter\n");
+   for (int i = 0; i < config->number_of_servers; i++)
+   {
+      data = pgmoneta_append(data, "pgmoneta_server_timeline{");
+
+      data = pgmoneta_append(data, "name=\"");
+      data = pgmoneta_append(data, config->servers[i].name);
+      data = pgmoneta_append(data, "\"} ");
+
+      data = pgmoneta_append_int(data, config->servers[i].cur_timeline);
+
+      data = pgmoneta_append(data, "\n");
+   }
+   data = pgmoneta_append(data, "\n");
+
+   data = pgmoneta_append(data, "#HELP pgmoneta_server_parent_tli The parent timeline of a timeline on a server\n");
+   data = pgmoneta_append(data, "#TYPE pgmoneta_server_parent_tli gauge\n");
+   for (int i = 0; i < config->number_of_servers; i++)
+   {
+      struct timeline_history* history = NULL;
+      struct timeline_history* curh = NULL;
+      int tli = 2;
+
+      data = pgmoneta_append(data, "pgmoneta_server_parent_tli{");
+
+      data = pgmoneta_append(data, "name=\"");
+      data = pgmoneta_append(data, config->servers[i].name);
+      data = pgmoneta_append(data, "\", ");
+
+      data = pgmoneta_append(data, "tli=\"");
+      data = pgmoneta_append_int(data, 1);
+      data = pgmoneta_append(data, "\"} ");
+
+      data = pgmoneta_append_int(data, 0);
```

Comment:
We should probably do `1` here

Reply:
Hmm, 0 means invalid tli, since timeline starts with 1. I think it's a good way to say timeline 1 doesn't have a parent.

Reply:
Ok